File: presence/concept-pl-execution-13-fixtures/concept_pulse.php
<?php
// concept: selective-retention
$signals = [2, 7, 1, 8, 2, 8];
$threshold = 5;
$scoreFloor = 100;
$retainedFloor = 3;
$score = 0; $retained = 0;
foreach ($signals as $index => $value) {
  $score += $value * ($index + 1);
  if ($value >= $threshold) $retained++;
}
$verdict = $score >= $scoreFloor && $retained >= $retainedFloor ? "coherent" : "fragmented";
echo "$verdict:$score:$retained\n";

File: presence/concept-pl-execution-13-fixtures/ConceptPulse.java
// concept: selective-retention
public final class ConceptPulse {
  public static void main(String[] args) {
    int[] signals = {2, 7, 1, 8, 2, 8};
    int threshold = 5;
    int scoreFloor = 100;
    int retainedFloor = 3;
    int score = 0; int retained = 0;
    for (int index = 0; index < signals.length; index++) {
      int value = signals[index]; score += value * (index + 1);
      if (value >= threshold) retained++;
    }
    String verdict = score >= scoreFloor && retained >= retainedFloor ? "coherent" : "fragmented";
    System.out.println(verdict + ":" + score + ":" + retained);
  }
}
